Orange/widgets/regression/owrandomforestregression.py

```diff
@@ -1,35 +1,24 @@
-from PyQt4 import QtGui
-from Orange.data import Table
-from Orange.regression.random_forest import (RandomForestRegressionLearner,
-                                             RandomForestRegressor)
-from Orange.widgets.classify.owrandomforest import OWRandomForest
+from Orange.regression.random_forest import RandomForestRegressionLearner
+from Orange.widgets.classify import owrandomforest
 from Orange.widgets import settings
 
 
-class OWRandomForestRegression(OWRandomForest):
+class OWRandomForestRegression(owrandomforest.OWRandomForest):
     name = "Random Forest Regression"
     description = "Random forest regression algorithm."
+    icon = "icons/RandomForest.svg"
+    priority = 40
 
     LEARNER = RandomForestRegressionLearner
-    outputs = [("Learner", LEARNER),
-               ("Model", RandomForestRegressor)]
+    OUTPUT_MODEL_NAME = "Model"
```

Comment:
We can just check `LEARNER` superclass. If `LEARNER` is a subclass of LearnerClassifier, `OUTPUT_MODEL_NAME = "Classifier"` and "Predictor" in the other case.